src/lib/providers/CurrencyProvider.tsx

```diff
@@ -29,14 +29,16 @@ const CurrencyProvider: React.FC<CurrencyProviderType> = ({ children }) => {
 	// Poll CoinGecko every 30 seconds
 	const getWildPriceUsd = async () => {
 		const price = await wildTokenPrice();
-		setWildPriceUsd(price || 0);
+		setWildPriceUsd(price);
+		if (price === 0) console.error('$WILD price API call failed');
 		await new Promise((resolve) => setTimeout(resolve, 30000));
 		getWildPriceUsd();
 	};
 
 	const getLootPriceUsd = async () => {
 		const price = await lootTokenPrice();
-		setLootPriceUsd(price || 0);
+		setLootPriceUsd(price);
+		if (price === 0) console.error('$LOOT price API call failed');
```

Comment:
You should be try/catching `await lootTokenPrice()` to see if it fails, not by checking if the price is zero.